simphony/cuds/tests/test_particles.py

```diff
@@ -61,29 +61,39 @@ class TestNativeContainerAddParticles(
         ContainerAddParticlesCheck, unittest.TestCase):
 
     def container_factory(self, name):
-        return ParticleContainer(name=name)
+        return Particles(name=name)
 
 
 class TestNativeContainerManipulatingParticles(
         ContainerManipulatingParticlesCheck, unittest.TestCase):
 
     def container_factory(self, name):
-        return ParticleContainer(name=name)
+        return Particles(name=name)
 
 
 class TestNativeContainerAddBonds(
         ContainerAddBondsCheck, unittest.TestCase):
 
     def container_factory(self, name):
-        return ParticleContainer(name=name)
+        return Particles(name=name)
 
 
 class TestNativeContainerManipulatingBonds(
         ContainerManipulatingBondsCheck, unittest.TestCase):
 
     def container_factory(self, name):
-        return ParticleContainer(name=name)
+        return Particles(name=name)
 
 
+class TestParticlesDataContainer(unittest.TestCase):
+
+    def test_data(self):
+        pc = Particles(name='foo')
+        data = pc.data
+        data[CUBA.MASS] = 9
+        pc.data = data
+        ret_data = pc.data
+        self.assertItemsEqual(data, ret_data)
```

Comment:
Please see related comments in https://github.com/simphony/simphony-common/pull/97#discussion_r26293247